### DISCUSSION TITLE
Document/Window/Tab title

### DISCUSSION THREAD

**Discussion by bogdanv-ntnu** [webmin/authentic-theme#1668] [General]
Hi
Is it possible to have the the hostname or ip of the host in the window/tab title instead?
Now is only Webmin/Dashboard + module blah blah .... can't identify the host so i have to go over each tab to find the one i want.
I think in the past it was possible but now i find this way of document.title useless. I am not interested to see what module i am currently using in the title. 
I know it's webmin since the icon is webmin. I see on the page what module i am using. 
I need to identify what server i wan to go to :)

**Comment by iliaross** [comment]
Hello,

Yeah, check this option in theme configuration page:

<img width="1207" alt="image" src="https://github.com/webmin/authentic-theme/assets/4426533/0836bfcb-e3a9-4a72-b3b0-c2fd2f2b8d2b">
